flex/views/experiment/TabSeqSetupView.mxml

```diff
@@ -105,64 +88,59 @@
 			}
 			updateLabelApplicationText();
 
-			// Only initialize once.
-//			if (themeList != null && codeRequestCategoryInitted == parentDocument.getRequestCategory().@codeRequestCategory
-//				&& seqPrepByCoreInitted == localPrepByCore && idLabInitted == parentDocument.getIdLab()) {
-//				return;
-//			}
-
 			// Only initialize the categories and answers for the "Select an sequencing experiment type" question
 			// if the state of the "Sample Details" tab (SampleSetupView) has changed.
-			if (parentDocument.sampleSetupView.currentState != previousSampleSetupState) {
-				previousSampleSetupState = parentDocument.sampleSetupView.currentState;
-				this.codeRequestCategoryInitted = parentDocument.getRequestCategory().@codeRequestCategory;
-				this.seqPrepByCoreInitted = localPrepByCore;
-				this.idLabInitted = parentDocument.getIdLab();
+			var codeRequestCategory:String = parentDocument.getRequestCategory().@codeRequestCategory;
+			if (codeRequestCategory != previousCodeRequestCategory) {
+				previousCodeRequestCategory = codeRequestCategory;
+				reloadApplications();
+			}
+		}
 
-				parentDocument.filteredAppList.refresh();
+		private function reloadApplications():void {
+			var done:Boolean = parentDocument.filteredAppList.refresh();
```

Comment:
Okay cool just wanted to be sure.  I'll wait for you to update that line and then I will merge it with master.  Thanks Matt.